docs/patterns/claim-check-content.md

```diff
@@ -10,7 +10,7 @@ Sending such large messages to the message bus directly is not recommended, beca
 
 ## Solution
 
-Store the entire message payload into an external service, such as a database. Get the reference to the stored payload, and send just that reference to the message bus. The reference acts like a claim check used to retrieve a piece of luggage, hence the name of the pattern. Clients interested in processing that specific message can use the obtained reference to retrieve the payload, if needed.
+There are two similar ways of handling this. The first is to store the entire message payload into an external service, such as a database, and then send just that stored payload reference to the message bus. The second is to store just the specific large-object, putting the reference to it in a message, which is then sent to the message bus. In both cases, the reference acts like a "claim check" used to retrieve a piece of luggage, hence the name of the pattern. Clients interested in processing that specific message can use the obtained reference to retrieve the payload, if needed.
```

Comment:
As this reads, it seems like a distinction without a difference, especially considering the last sentence which is "Clients interested in processing that specific message can use the obtained reference to retrieve the payload"

What's the distinction you're trying to make here, and can you be more specific?  For this being the opening paragraph basically, I'd rather not have it more complex than necessary to teach the topic. Maybe this nuance (if you can describe it more completely) can go later in the article?

Reply:
The thing is when combining this pattern with other messaging patterns, specifically routing (whether based on either the "type" of the message or any other of its contents), then storing the entire message payload into blob storage isn't appropriate.

Reply:
Can you come up with some new, more distinctive text to help bring this point out?  As it reads now, it reads as a distinction without a difference.  Especially being the FIRST sentence in the solution.  I wonder if it would be better to lead with one way, and then talk about a similar, but alternative approach and what use case it's suited for.  It's just weird to have "the solution" section lead with "two ways" as the first sentence -- and the two ways aren't very distinguised, as written.

So, maybe seperate them out and expand on the thoughts a bit more?